docs/faq.md

```diff
@@ -12,7 +12,7 @@ Since Test Pilot is an open source project, you can see all of the cool ways we
 
 ## Why isn't a Test Pilot icon showing up in my toolbar?
 
-Test Pilot can't be installed in [private browsing mode](https://support.mozilla.org/en-US/kb/private-browsing-use-firefox-without-history?redirectlocale=en-US&redirectslug=Private+Browsing), 
```

Comment:
DId you really want to remove the trailing space? IIRC it's significant for new lines in markdown.